scripts/linux/runE2ETest.sh

```diff
@@ -1001,6 +1016,9 @@ function usage() {
     echo ' -logAnalyticsWorkspaceId        Optional Log Analytics workspace ID for metrics collection and reporting.'
     echo ' -logAnalyticsSharedKey          Optional Log Analytics shared key for metrics collection and reporting.'
     echo ' -logAnalyticsLogType            Optional Log Analytics log type for the Analyzer module.'
+    echo ' -metricsEndpointsCSV            Optional csv of exposed endpoints for which to scrape metrics.'
+    echo ' -metricsScrapeFrequencyInSecs   Optional frequency at which the MetricsCollector module will scrape metrics from the exposed metrics endpoints. Default is 300 seconds.'
+    echo ' -metricsUploadTarget            Optional upload target for metrics. Valid values are AzureLogAnalytics or IoTHub. Default is AzureLogAnalytics.'
```

Comment:
Hmmm... this metrics are technically required if the metric in EdgeHub is enable.
Maybe it's time to do some Bash gymnastic similar to this ? 
https://github.com/and-rewsmith/iotedge/blob/7ec1ce1b1303ca875d92a5e04a02b87e1e995d1e/scripts/linux/runE2ETest.sh#L446-L449

Reply:
Great idea!

Reply:
Bear and I discussed in person and since log analytics should always be enabled, we should remove logAnalyticsEnabled field from the analyzer and the pipeline.

Reply:
However, we chose to not address the field in this PR.

Reply:
Philip confirmed that we should not change so I am resolving